resources/js/mosaic.layout.js

```diff
@@ -1259,7 +1260,10 @@ export default class LayoutManager {
             // Select new tile and make it draggables
             if (new_tile && original_tile.length > 0) {
                 original_tile.mosaicAddDrag();
-                original_tile.data("mosaic-tile").initializeContent();
+                var tiletype = original_tile.data("mosaic-tile").getType();
+                if (tiletype === "plone.app.standardtiles.html") {
+                    original_tile.data("mosaic-tile").setupWysiwyg();
+                }
```

Comment:
I would move this to the `initializeContent()` method of the tile API. There are the type switches already.